pkgs/os-specific/linux/numactl/default.nix

```diff
@@ -26,6 +28,13 @@ stdenv.mkDerivation rec {
     description = "Library and tools for non-uniform memory access (NUMA) machines";
     homepage = "https://github.com/numactl/numactl";
     license = with licenses; [ gpl2 lgpl21 ]; # libnuma is lgpl21
-    platforms = [ "i686-linux" "x86_64-linux" "aarch64-linux" ];
+    platforms = [
+      "i686-linux"
```

Comment:
Maybe we can set this to just linux?

Reply:
Makes sense to me. I'm not sure what the motivation was for listing the platform doubles explicitly here.